### DISCUSSION TITLE
How to send custom request to GitHub API using the library

### DISCUSSION THREAD

**Discussion by lmmendes** [hub4j/github-api#1841] [Q&A] [answered]
I'm trying to get the [allowed actions](https://docs.github.com/en/rest/actions/permissions?apiVersion=2022-11-28#get-allowed-actions-and-reusable-workflows-for-a-repository) for a given repository, but this seems to be unsupported via the current version of the client.

Is there any "official" or suggested way to get access to the `createRequest` method or similar so that I can make custom calls to the GitHub API with the current instance of the client?

**Comment by bitwiseman** [comment, marked as answer]
@lmmendes  It is not currently supported - `createRequest` is an implementation detail that has changed quite a bit.  Thoughts around this issue are tracked in #513 .
